### DISCUSSION TITLE
How can I export all IOCs from misp to a text file ?

### DISCUSSION THREAD

**Discussion by gopala410** [MISP/PyMISP#738] [General]
Hi, 
Id like to export all IOCs from misp to a text file. 
Is it possible to do so ? 
Id like to do it separating the IOCs files in different text files. 
One txt file for IPs, another different txt file for domains,,,,,,, etc...

**Comment by Rafiot** [comment]
Hi, you can use the search query: https://pymisp.readthedocs.io/en/latest/modules.html#pymisp.PyMISP.search

**Reply by gopala410** [reply]
How can I use this search query ? I have no idea howto
Im a newby Im kinda lost

**Reply by hkelley** [reply]
Do you need PyMISP at all?  Perhaps the default /download URLs would suffice, e.g.

https://<misp url>/attributes/text/download/ip-src

The full list is here.  [https://misp.gitbooks.io/misp-book/content/automation/#text-export](https://misp.gitbooks.io/misp-book/content/automation/#text-export)
